code/modules/assembly/flash.dm

```diff
@@ -228,24 +233,32 @@
 		return
 	AOE_flash()
 
+/**
+  * Converts the victim to revs
+  *
+  * Arguments:
+  * * M - Victim
+  * * user - Attacker
+  */
 /obj/item/assembly/flash/proc/terrible_conversion_proc(mob/living/carbon/H, mob/user)
```

Comment:
Since this is the only place we use this proc and we're the only people who can use it, we can rename these vars properly.

Perhaps (mob/living/carbon/victim, mob/attacker) would make the code more readable? They're almost self-documenting then. Still need 'em DMDoc'ed though so you can't delete that.